test/shared/src/main/scala/zio/test/TimeVariants.scala

```diff
@@ -16,13 +16,29 @@
 
 package zio.test
 
+import zio.ZIO
 import zio.duration.Duration
 import zio.random.Random
+import zio.stream.ZStream
 
-import java.time.{Instant, LocalDateTime, OffsetDateTime, ZoneOffset}
+import java.time._
 
 trait TimeVariants {
 
+  /**
+   * A generator of `java.time.DayOfWeek` values. Doesn't have any shrinking.
```

Comment:
This will shrink towards the first value, in this case `MONDAY`, which I think is fine.

Reply:
Right! I've taken a look at `DayOfWeek` values and indeed `MONDAY` goes first. That's why I put those in the same order.

Reply:
So it makes sense to shrink toward `MONDAY`!